firstvoices/backend/serializers/character_serializers.py

```diff
@@ -0,0 +1,55 @@
+from rest_framework import serializers
+
+from backend.models.characters import Character, CharacterVariant, IgnoredCharacter
+from backend.models.dictionary import DictionaryEntry
+from backend.serializers.dictionary_serializers import DictionaryEntryDetailSerializer
+from backend.serializers.fields import SiteHyperlinkedIdentityField
+
+
+class IgnoredCharacterSerializer(serializers.ModelSerializer):
+    url = SiteHyperlinkedIdentityField(view_name="api:ignored_characters-detail")
+    site = serializers.StringRelatedField()
+
+    class Meta:
+        model = IgnoredCharacter
+        fields = ["url", "id", "title", "site"]
```

Comment:
Minor: I don't think we need the site here, since the only way to retrieve this info is via a site-specific url.

(The dictionary entry response includes the site because eventually you'll be able to get that info via an app-level search box that includes entries from all sites.)